modules/cpmso-feat-auto-update.adoc

```diff
@@ -2,12 +2,55 @@
 = Automatic updates to the control plane configuration
+//Not for ROSA/OSD:
+ifndef::openshift-dedicated,openshift-rosa[]
+The `RollingUpdate` update strategy automatically propagates changes to your control plane configuration.
+This update strategy is the default configuration for the control plane machine set.
+endif::openshift-dedicated,openshift-rosa[]
+//For ROSA:
+ifdef::openshift-rosa[]
+On ROSA (classic architecture) clusters, control plane machine sets automatically propagate changes to your control plane configuration.
+When a control plane machine needs to be replaced, the Control Plane Machine Set Operator creates a replacement machine based on the configuration specified by the `ControlPlaneMachineSet` custom resource (CR). Once the new control plane machine is ready, the Operator safely drains and terminates the old control plane machine in a way that mitigates any potential negative effects on cluster API or workload availability.
+endif::openshift-rosa[]
+//For OSD:
+ifdef::openshift-dedicated[]
+On {product-title} clusters, the control plane machine sets automatically propagate changes to your control plane configuration.
+When a control plane machine needs to be replaced, the Control Plane Machine Set Operator creates a replacement machine based on the configuration specified by the `ControlPlaneMachineSet` custom resource (CR). Once the new control plane machine is ready, the Operator safely drains and terminates the old control plane machine in a way that mitigates any potential negative effects on cluster API or workload availability.
+endif::openshift-dedicated[]
+//Not for ROSA/OSD:
+ifndef::openshift-dedicated,openshift-rosa[]
+
+For clusters that use the `RollingUpdate` update strategy, the Operator creates a replacement control plane machine with the configuration that is specified in the CR.
+
+When the replacement control plane machine is ready, the Operator deletes the control plane machine that is marked for replacement because the machine is either out of spec or in an unhealthy state.
+The replacement machine then joins the control plane.
+
+If multiple control plane machines are marked for replacement, the Operator protects etcd health during replacement by repeating this replacement process one machine at a time until it has replaced each machine.
+endif::openshift-dedicated,openshift-rosa[]
+
+ifdef::openshift-dedicated,openshift-rosa[]
+A control plane machine can be marked for replacement at any time, typically because the machine has fallen out of spec or entered an unhealthy state. Such replacements are a normal part of a cluster's lifecycle and not a cause for concern. SRE will be alerted to the issue automatically if any part of a control plane node replacement fails.
 
-The `RollingUpdate` update strategy automatically propagates changes to your control plane configuration. This update strategy is the default configuration for the control plane machine set.
+[NOTE]
```

Comment:
Try and avoid stacked admonitions:

![Screenshot from 2024-07-05 14-10-41](https://github.com/openshift/openshift-docs/assets/57954076/3c99b035-4b3a-47c4-a01d-38c8096490ee)

Reply:
Maybe move the important admonition closer to the top where it would hold more value?

Reply:
> Maybe move the important admonition closer to the top where it would hold more value?

Thanks @dfitzmau. I did think of doing this but thought the information in the first admonition was better placed after the opening paragraph but you're right, they don't look great stacked on top of each other. I will revisit this!